webhook_features/src/features/summary_comment.rs

```diff
@@ -314,87 +312,22 @@ struct Review {
     date: chrono::DateTime<chrono::Utc>,
 }
 
-fn is_commit_hash(s: &str) -> bool {
-    // Use length from https://github.com/bitcoin-core/bitcoin-maintainer-tools/blob/78ab16ae88af7a5ef886ae8cef6df2e9ef3f6085/github-merge.py#L211
-    s.len() >= 6 && s.chars().all(|c| c.is_ascii_hexdigit())
-}
-
 #[derive(Debug, PartialEq)]
 struct AckCommit {
     ack_type: AckType,
     commit: Option<String>,
 }
 
 fn parse_review(comment: &str) -> Option<AckCommit> {
-    let comment = comment.to_lowercase();
-    let words = comment
-        .split('\n')
-        .filter(|s| !s.starts_with('>')) // Ignore quoted text
```

Comment:
I think you can keep this pre-processing and drop the `m` flag. A review should never span multiple lines, right?

Reply:
Sure, will do

Reply:
Done https://github.com/MarcoFalke/DrahtBot/compare/e3ec8ebd735522c3f5064e0ed8aff91e04220aff..680d12b59e43dda1a42b93b0b2e67bf1efa47394